### DISCUSSION TITLE
Does NOT download Entware files

### DISCUSSION THREAD

**Discussion by slonce70** [Guilouz/Creality-Helper-Script-Wiki#510] [Issues]
Hey, everybody. Printer k1c - latest firmware installed. I rooted the printer 2 weeks ago and everything was fine. Now also installed - everything installs, all scripts are installed but does not download entware - just hangs on download and that's it. What can be the problem!? Reset has already done 2 times. The problem remains.

I am from Ukraine and it turns out the main domain does not work but the secondary domain works - but it does not connect to it and tries to download from the main domain.

![Знімок екрана 2024-04-05 233424](https://github.com/Guilouz/Creality-K1-Series/assets/130596182/68affd96-8bbd-4fef-be4f-7bea2a00fec7)
![Знімок екрана 2024-04-05 233916](https://github.com/Guilouz/Creality-K1-Series/assets/130596182/d940becc-8ba9-4802-a30f-93fce5be5724)

**Comment by Guilouz** [comment]
Make sure you have access to this URL on your network: https://bin.entware.net/mipselsf-k3.4/

**Comment by kuubek55** [comment]
> Hey, everybody. Printer k1c - latest firmware installed. I rooted the printer 2 weeks ago and everything was fine. Now also installed - everything installs, all scripts are installed but does not download entware - just hangs on download and that's it. What can be the problem!? Reset has already done 2 times. The problem remains.
> 
> I am from Ukraine and it turns out the main domain does not work but the secondary domain works - but it does not connect to it and tries to download from the main domain.
> 
> ![Знімок екрана 2024-04-05 233424](https://private-user-images.githubusercontent.com/130596182/320131997-68affd96-8bbd-4fef-be4f-7bea2a00fec7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2MjYxNTQsIm5iZiI6MTcxMjYyNTg1NCwicGF0aCI6Ii8xMzA1OTYxODIvMzIwMTMxOTk3LTY4YWZmZDk2LThiYmQtNGZlZi1iZTRmLTdiZWEyYTAwZmVjNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwOVQwMTI0MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYWU1Y2E4OTE5MTk5Nzk5NWNiOGQ2NTY5OWVkMTFkMTVkNDkzNzc1ODQ2ZDY3NjlmMDZhMjg2YTUwZTZlNDAzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.0yhffUr21H4ESoHDZe5xoPulWos-y99Pkk4tvMGfHKU) ![Знімок екрана 2024-04-05 233916](https://private-user-images.githubusercontent.com/130596182/320133015-d940becc-8ba9-4802-a30f-93fce5be5724.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2MjYxNTQsIm5iZiI6MTcxMjYyNTg1NCwicGF0aCI6Ii8xMzA1OTYxODIvMzIwMTMzMDE1LWQ5NDBiZWNjLThiYTktNDgwMi1hMzBmLTkzZmNlNWJlNTcyNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwOVQwMTI0MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NTJmZTQ4YjY1MjIwMTdkMTRmMDY1YTJmYjYwY2MxNDcxOTAzMmJmZjhlYzdjYjVmMmZlNWE0NTM1YTY2ZTAxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.A3lNmiva4lxm8oQCu66cH5duEBNGRDmR5Y0K4iHxx_k)

Same thing for me, I used VPN on my router

**Comment by djjuno** [comment]
@Guilouz The main url that you use for Entware has been broken for a while now. https://bin.entware.net/mipselsf-k3.4/ no longer exists. If you edit the script specifically for entware then comment out https://bin.entware.net/mipselsf-k3.4/ and uncomment the second address it will download. I was hoping that someone else would mention this. It's not been working for me for a while now. That was the workaround that I found.

**Reply by alvaropsouza** [reply]
Thank you, worked for me

**Comment by djjuno** [comment]
This is off a fresh install that I just did about 5 minutes ago.

![image](https://github.com/Guilouz/Creality-K1-Series/assets/24688466/96bce0ce-002d-4b2a-9b35-4854d2e769c8)

If you uncomment "#URL="http://www.openk1.org/static/entware/mipselsf-k3.4/installer" and comment URL="https://bin.entware.net/mipselsf-k3.4/installer" then run the helper script and install Entware. It will install. I think the link for the site is just broken in general.

echo -e "Info: Downloading opkg package manager..."
chmod 755 /usr/data/helper-script/files/fixes/curl
URL="https://bin.entware.net/mipselsf-k3.4/installer"
#URL="http://www.openk1.org/static/entware/mipselsf-k3.4/installer"
/usr/data/helper-script/files/fixes/curl -L "$URL/opkg" -o "/opt/bin/opkg"
/usr/data/helper-script/files/fixes/curl -L "$URL/opkg.conf" -o "/opt/etc/opkg.conf"

**Comment by djjuno** [comment]
Here are the results once the change is made:

![image](https://github.com/Guilouz/Creality-K1-Series/assets/24688466/a6b6fd9d-ee44-4976-9bc4-18b1f816274e)
![image](https://github.com/Guilouz/Creality-K1-Series/assets/24688466/2443aae1-1d5f-41db-b86a-cc605f547158)
